### PR TITLE
Add metrics to build page

### LINKED ISSUE #343: Builds should show their pod metrics if they can
It's frustrating not seeing them.  However, note that for builds the prometheus query is for pod_name="X",container_name="" (we have to show the full pod CPU because the build is run under the pod cgroup, not a container cgroup).

### PR DESCRIPTION
**Pull request by spadgett**
@smarterclayton 

bin-cross details openshift origin 2018-07-31 15-45-45

Closes #343

### CODE REVIEW

**Comment by openshift-docker** [comment]
lgtm thank you

**Comment by spadgett** [comment]
jenkins rebuild

**Comment by spadgett** [comment]
jenkins rebuild

File: frontend/public/components/build.tsx [hunk 1]
```diff
@@ -42,6 +43,29 @@ export enum BuildStrategyType {
...
+      </div>
+      <div className="col-md-4">
+        <Line title="CPU Shares" query={`pod_name:container_cpu_usage:sum{pod_name='${podName}',container_name='',namespace='${build.metadata.namespace}'} * 1000`} />
+      </div>
+      <div className="col-md-4">
+        <Line title="Filesystem (bytes)" query={`pod_name:container_fs_usage_bytes:sum{pod_name='${podName}',container_name='',namespace='${build.metadata.namespace}'}`} />
+      </div>
+    </div>
+
+    <br />
```
Comment (rhamilto) [inline frontend/public/components/build.tsx]: Nit:  this  makes me wince.  No way to fix with CSS?

Comment (spadgett) [reply frontend/public/components/build.tsx]: We could. I was following what was there for pods and nodes. For instance:

https://github.com/openshift/console/blob/master/frontend/public/components/pod.jsx#L151

We'd probably want to fix it in all places.

**Pull request merged**